### PR TITLE
chore(generate): generate projects with a more consistent JavaScript syntax

### PR DESCRIPTION
**Pull request by arnaudbesnier**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Create automatic tests
- [x] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

File: templates/app/app.hbs [hunk 1]
```diff
@@ -48,13 +48,13 @@ app.use('/forest', (request, response, next) => {
 requireAll({
   dirname: path.join(__dirname, 'routes'),
   recursive: true,
-  resolve: Module => app.use('/forest', Module),
+  resolve: (Module) => app.use('/forest', Module),
```
Comment (rap2hpoutre) [inline templates/app/app.hbs:51]: [side note] I'm not sure we are consistent in our own code! 😅 (lint rules differ: sometimes I have to remove the parenthesis)

File: templates/app/middlewares/forestadmin.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@ const Liana = require('forest-express-sequelize');
 const { sequelize } = require('../models');
 {{/if}}
 
-module.exports = async function (app) {
+module.exports = async function forestadmin(app) {
```
Comment (rap2hpoutre) [inline templates/app/middlewares/forestadmin.hbs:11]: Named functions are way better 👍

File: templates/app/app.hbs (hunk 1 above)
Comment (arnaudbesnier) [reply templates/app/app.hbs:51]: Really, are you sure about that? I don't know such scenarios.
I'd love to have the opportunity to make the rules more consistent, feel free to share those inconsistent cases.

Comment (rap2hpoutre) [reply templates/app/app.hbs:51]: @arnaudbesnier Yes, sorry about being vague 😅 ! If you look at **Forestadmin** repository source code, you should see 300+ occurrences of `x => y` without parenthesis.

Capture d’écran 2020-08-13 à 09 03 19

If you try to add a parenthesis, you should see this linter error:

Capture d’écran 2020-08-13 à 09 04 56

Still, this rule differs in **Forestadmin server**.

**Pull request merged**